packages/viz/src/lib/style/default-styles.ts

```diff
@@ -1,53 +1,121 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// TODO remove any
 import { hexToRgb } from './helpers/utils';
 import { GeometryType } from '../sources/Source';
 
-const pointStyles = {
-  opacity: 1.0,
-  stroked: true,
-  filled: true,
+const defaults: any = {
+  Point: {
+    color: '#EE4D5A',
```

Comment:
I think we're mixing here default styles with properties that just apply to (some) style helpers. I find it quite awkward. 

IMO those extra 'default' params should be in the style helpers themselves. If we ever need to change something in them, for example the name of a parameter (sizeRange, to size) we would need to change in 2 places: the helper and here

Reply:
Think about the following use case:  the palette property is defined in ColorCategories, ColorBins and ColorContinuous.

The idea behind this is: the const `defaults` at `default-styles` we defined all the styles related with layout that are reused in several components. 

That's true I didn't do it with sizeRange, Just fixed 😄